src/services/bindService.js

```diff
@@ -74,6 +74,85 @@ angular.module("openshiftCommonServices")
       return binding;
     };
 
+    var isServiceBindable = function(serviceInstance, serviceClasses) {
```

Comment:
Do you mind adding a check for `metadata.deletionTimestamp` while you're here?

https://github.com/openshift/origin-web-console/issues/1874